{
	"title": "CVE-2014-1604",
	"cve": "CVE-2014-1604",
	"notes": [
		{
			"type": "Description",
			"text": "The parser cache functionality in parsergenerator.py in RPLY (aka python-rply) before 0.7.1 allows local users to spoof cache data by pre-creating a temporary rply-*.json file with a predictable name."
		},
		{
			"title": "Published",
			"type": "Other",
			"text": "2014-01-27"
		},
		{
			"title": "Modified",
			"type": "Other",
			"text": "2017-08-28"
		}
	],
	"references": [
		{
			"url": "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=735263",
			"description": "CONFIRM:http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=735263"
		},
		{
			"url": "https://github.com/alex/rply/commit/fc9bbcd25b0b4f09bbd6339f710ad24c129d5d7c",
			"description": "CONFIRM:https://github.com/alex/rply/commit/fc9bbcd25b0b4f09bbd6339f710ad24c129d5d7c"
		},
		{
			"url": "http://www.openwall.com/lists/oss-security/2014/01/17/8",
			"description": "MLIST:[oss-security] 20140114 Fwd: [Python-modules-team] Bug#735263: python-rply: insecure use of /tmp"
		},
		{
			"url": "http://www.openwall.com/lists/oss-security/2014/01/18/4",
			"description": "MLIST:[oss-security] 20140117 Re: Fwd: [Python-modules-team] Bug#735263: python-rply: insecure use of /tmp"
		},
		{
			"url": "http://www.osvdb.org/102202",
			"description": "OSVDB:102202"
		},
		{
			"url": "http://secunia.com/advisories/56429",
			"description": "SECUNIA:56429"
		},
		{
			"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/90593",
			"description": "XF:rply-cve20141604-insecure-permissions(90593)"
		}
	]
}
